src/app.ts

```diff
@@ -32,6 +32,14 @@ prClient.getPullRequests().then(prs => {
         const tableCellRepo = document.createElement("td");
         tableCellRepo.innerText = pr.repo;
         tableRow.appendChild(tableCellRepo);
+        // Base cell
+        const tableCellBaseBranch = document.createElement("td");
+        tableCellBaseBranch.innerHTML = "<a href='" + (pr.baseUri + pr.projectName + "/_git/" + pr.repo + "?version=GB" + pr.baseBranch) + "' target='_top'>#" + pr.baseBranch + "</a>";
```

Comment:
While this _appears_ to be right, does anybody know what this GB prefix is about? Is there any chance of some users having a different prefix?

Reply:
Worst case the new feature does not work for them I suppose. I don't see how it could do harm.

Reply:
Terrific question.  I'm guessing the "GB" stands for Git branch?  I'm guessing it's an identifier to tell the DevOps back end that the request in question is for a Git repo and not a TFS... uh... Source Control... thing.